docs/framework/network-programming/changes-to-the-system-uri-namespace-in-version-2-0.md

```diff
@@ -50,9 +50,9 @@ Several changes were made to the <xref:System.Uri?displayProperty=nameWithType>
   
 -   <xref:System.Uri.IsLoopback%2A> now produces consistent results.  
   
--   The URI "`file:///path`" is no longer translated into "file://path".  
+-   The URI "`file:///path`" is no longer translated into `file://path`.  
```

Comment:
should you keep the quotes as the first one also has them? or remove from both?

Reply:
Either way. I just tried to fix the issue, not redesign it 😄